x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/index.tsx

```diff
@@ -312,21 +314,42 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
                 {configName || configId}
               </EuiLink>
             </EuiFlexItem>
-            <EuiFlexItem grow={true}>
-              <EuiText color="subdued" size="xs" style={{ whiteSpace: 'nowrap' }}>
-                <FormattedMessage
-                  id="xpack.ingestManager.agentList.revisionNumber"
-                  defaultMessage="rev. {revNumber}"
-                  values={{ revNumber: '999' }} // TODO fix when we have revision
-                />
-              </EuiText>
-            </EuiFlexItem>
+            {agent.config_revision && (
+              <EuiFlexItem grow={false}>
+                <EuiText color="default" size="xs" style={{ whiteSpace: 'nowrap' }}>
```

Comment:
Can use [EUI utility class](https://elastic.github.io/eui/#/utilities/css-utility-classes) `className="eui-textNoWrap"` instead of inline style here